File: api/MyPasswordAPI.Domain/Validators/PasswordValidator.cs
using FluentValidation;
using MyPasswordAPI.Domain.Entities;

namespace MyPasswordAPI.Domain.Validators
{
    public class PasswordValidator: AbstractValidator<Password>
    {
        public PasswordValidator()
        {
            RuleFor(v => v.CustomerId)
                .NotNull()
                .WithMessage("* O ID do usuário não pode ser nulo.");

            RuleFor(v => v.Title)
                .NotNull()
                .WithMessage("* O Título da senha não pode ser nulo.")

                .NotEmpty()
                .WithMessage("* O Título da senha não pode ser vazio.")
                
                .MinimumLength(3)
                .WithMessage("* O Título da senha deve ter no mínimo 3 caracteres.")

                .MaximumLength(40)
                .WithMessage("* O Título da senha deve ter no máximo 40 caracteres.");


            RuleFor(v => v.Value)
                .NotNull()
                .WithMessage("* O Valor da senha não pode ser nulo.")

                .NotEmpty()
                .WithMessage("* O Valor da senha não pode ser vazio.")

                .MaximumLength(400)
                .WithMessage("* O Valor da senha deve ter no máximo 400 caracteres.");
        }
    }
}

File: app/lib/models/validators/password.validator.dart
import 'package:mypassword/models/entities/password.model.dart';
import 'package:mypassword/models/enums/inputType.enum.dart';
import 'package:mypassword/models/validators/errorsValidation.model.dart';
import 'base.validator.dart';

class PasswordValidator extends BaseValidator<Password>{
	
  @override
  void validate(Password password){
    
    //customerID
    if(password.customerId == null)
      errors.add(new ErrorsValidation(inputType: InputType.customerId, error: "O ID do usuário não pode ser nulo."));

    //title
    if(password.title == null || password.title == "")
      errors.add(new ErrorsValidation(inputType: InputType.title, error: "O título da senha não pode ser vazio."));

    if(password.title.length < 3 || password.title.length > 40)
      errors.add(new ErrorsValidation(inputType: InputType.title, error: "O título da senha deve ter entre 3 e 40 caracteres."));

    //value
    if(password.value == null || password.value == "")
      errors.add(new ErrorsValidation(inputType: InputType.value, error: "O valor da senha não pode ser vazio."));

	 if(password.value.length > 400)
	 	errors.add(new ErrorsValidation(inputType: InputType.value, error: "O valor da senha deve ter no máximo 400 caracteres."));
  }
}